stdlib/private/SwiftPrivateThreadExtras/CMakeLists.txt

```diff
@@ -10,14 +10,14 @@ add_swift_target_library(swiftSwiftPrivateThreadExtras ${SWIFT_STDLIB_LIBRARY_BU
   SWIFT_MODULE_DEPENDS_OSX Darwin
   SWIFT_MODULE_DEPENDS_TVOS Darwin
   SWIFT_MODULE_DEPENDS_WATCHOS Darwin
+  SWIFT_MODULE_DEPENDS_FREESTANDING Darwin
```

Comment:
This may not always be true (considered free standing on Linux will want Glibc here and on windows would want CRT and WinSDK). I imagine that instead the right thing to do is make this a cmake variable.